stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -216,6 +220,14 @@
 ;;   for the given reward cycle
 (define-map aggregate-public-keys uint (buff 33))
 
+;; Data var used to track signer set for current cycle
+(define-data-var current-signer-set (list 4096 (buff 33)) (list ))
+
+;; Map that tracks the signing-key & cycle to a list of key-ids
+(define-map reward-cycle-signing-key-ids 
+    {cycle: uint, signer: (buff 33)}
+    {key-ids: (list 4096 uint)})
```

Comment:
I think it should be only 4000 key ids (same number of reward slots at least which I think is 4000? but maybe its 4096?)

Reply:
Yeah unless we've upped the number of reward slots?

Reply:
Copy. Thought I heard 4096 in one of the latest meetings but will change both to 4000.

Reply:
This value is different on testnet (1000) and we can't use a constant :-/

Reply:
I think the 4096 is the number of slots total that can be in a given stacker-db instance. :)

Reply:
Double-checking here:

1. The max list size for the signer set should be 4096*
2. The max list size for the key-ids should be 4000*

Correct?

Reply:
They need to be the same value.  I'm fine with it being 4000.